fiscalyear.py

```diff
@@ -225,11 +230,9 @@ def __contains__(self, item):
             return self == item
         elif isinstance(item, FiscalQuarter):
             return self._fiscal_year == item.fiscal_year
-        elif (isinstance(item, FiscalDateTime) or
-              isinstance(item, datetime.datetime)):
+        elif isinstance(item, datetime.datetime):
             return self.start <= item <= self.end
-        elif (isinstance(item, FiscalDate) or
-              isinstance(item, datetime.date)):
+        elif isinstance(item, datetime.date):
```

Comment:
This is because `FiscalDate` is a subclass of `datetime.date` so there's no need to check both?

Reply:
Yeap!